### ISSUE TITLE
Reconnecting-websocket options not working?

### ISSUE THREAD

**Issue by dasbts** [ViewBlock/binance-api-node#635]
Hi, 

I'm having the same issue as all the closed reconnecting-websocket problems, and in https://github.com/ViewBlock/binance-api-node/issues/621 it was mentioned to pass options. I investigated and could not find any possible way to add options to miniticker. Is this something that can be looked into? 

I checked at node_modules/binance-api-node/index.d.ts and (correct me if I'm wrong but) the code as far as I understand it takes a symbol pair, and then you choose where to get the callback to. 
```
miniTicker: (
      pair: string | string[],
      callback: (ticker: MiniTicker) => void,
    ) => ReconnectingWebSocketHandler
``` 

If there is some way to add options such as connectionTimeout to the miniticker already that I don't know about?

**Comment by dasbts** [comment]
Still wondering about this one, tried so many ways. Anyone know? @balthazar previously refered to using connectionTimeout but idk how to pass it in. Had half a year of disconnecting websockets. :(

**Comment by nfjdu** [comment]
Agree. After internet drop for ~5minutes i got ws closed. And there is no way to easily reopen it, rather then write own interval 'pinging' waiting for internet to restore, then closing/reopening ws, or something like that.  

https://github.com/balthazar/binance-api-node/issues/621#issuecomment-1595565823
Talking about this comment - i dont see the ability to tweak `connectionTimeout`, and im not sure it would even help. What is the problem with giving user ability to listen for ws events?
